PhotoBooth/Camera Page/CameraViewController.swift

```diff
@@ -172,9 +179,51 @@ class CameraViewController: UIViewController {
             }
         }
     }
-    
+
+    @objc func flashMode() {
+        switch viewModel.flashType {
+        case .on:
+            UIView.animate(withDuration: 0.1,
+                           animations: {
+                            self.flashButton.transform = CGAffineTransform(scaleX: 0.6, y: 0.6)
+                            self.viewModel.toggleFlash()
+            },
+                           completion: { _ in
+                            UIView.animate(withDuration: 0.1) {
+                                print(self.viewModel.captureSession.getFlashMode().rawValue)
+                                self.flashButton.setImage(UIImage(named: "flash_off"), for: .normal)
+                                self.flashButton.transform = CGAffineTransform.identity
+                            }
+            })
+        case .off:
+            UIView.animate(withDuration: 0.1,
+                           animations: {
+                            self.flashButton.transform = CGAffineTransform(scaleX: 0.6, y: 0.6)
+                            self.viewModel.toggleFlash()
+            },
+                           completion: { _ in
+                            UIView.animate(withDuration: 0.1) {
+                                print(self.viewModel.captureSession.getFlashMode().rawValue)
+                                self.flashButton.setImage(UIImage(named: "flash_on"), for: .normal)
+                                self.flashButton.transform = CGAffineTransform.identity
+                            }
+            })
+        default:
```

Comment:
what about `.auto`?

Reply:
you said we weren't going with auto. you said to make it like snapchats

Reply:
but you still have auto as an option

Reply:
you mean in the set flash options?

Reply:
yeah